### DISCUSSION TITLE
Context Option "Open With..." wasn't removed after app deletion.

### DISCUSSION THREAD

**Discussion by Eshku** [phcode-dev/phoenix#1903] [Q&A] [answered]
I deleted Phoenix app, but option in the context menu is still there.
Resstarting PC doesn't help, how do I get rid of it (❁´◡`❁)

![image](https://github.com/user-attachments/assets/51c02b20-4feb-4e9b-8b66-319fc4ac232d)

**Comment by abose** [comment]
@charlypa the context menu is not getting removed on uninstall!

**Reply by charlypa** [reply, marked as answer]
@Eshku 
It seems like your installation or the installer might be corrupted. Could you try downloading the latest installer from [phcode.io](https://phcode.io), then proceed with an uninstall? Let me know if that resolves the issue.

**Comment by Eshku** [comment]
Yep it worked properly after reinstall, I guess something broke on my end ¯\_(ツ)_/¯
